languages/tree-sitter-stack-graphs-java/src/stack-graphs.tsg

```diff
@@ -155,25 +189,165 @@ attribute node_symbol = node            => symbol = (source-text node), source_n
 ;;  Member Declarations
 ;; =====================
 
+[
+  (class_declaration)
+  (enum_declaration)
+  (field_declaration)
+  (interface_declaration)
+  (method_declaration)
+  (constructor_declaration)
+  (annotation_type_declaration)
+  (constant_declaration)
+  (record_declaration)
+] @decl
```

Comment:
As currently defined, the Java grammar sometimes omits clear, visible syntactic categories like _declaration_, _statement_, _expression_, _type_, etc. So we list them.

It might be nice to have query macros so I could write `$declaration = [ … ]` and then use `$declaration` to match within other queries.